### DISCUSSION TITLE
Dados acumulados dia 02-11-2020

### DISCUSSION THREAD

**Discussion by hiimbia** [dssg-pt/covid19pt-data#1210] [Dados por Concelho]
Bom dia,
estou à procura dos dados acumulados para casos confirmados e dos óbitos no dia 02-11-2020. Vi que têm o repository concelhos_new mas esse não inclui esta data especifica e o repository anterior também só vai até 26-10. 
Se me puderem ajudar, agradecia imenso!
Obrigada

**Comment by davipt** [comment]
Assumo que esteja à procura de dados por concelho.

A DGS deixou de publicar dados acumulados para concelhos no relatório correspondente a 26-10-2020.

A partir daí só publicaram dados de incidência a 14 dias - casos e óbitos nos 14 dias anteriores à data reportada, por cem mil habitantes.

São esses dados que aparecem no `concelhos_new` (um ficheiro temporário que ficou-se por lá) e transformados nos valores correspondentes de incidencia no `concelhos_incidencia`, e convertidos em valores relativos no `concelhos_14dias`.

Em teoria deveria ser possível derivar os valores absolutos mediante a analise dos valores a 14 dias e os ultimos totais disponíveis, mas como na altura já era conhecido que os dados absolutos por concelho não estavam correctos, optou-se por publicar apenas os dados novos nos ficheiros novos.

Muitos dos dados são actualizados retroactivamente. Por exemplo pode haver um caso reportado num concelho, por ser feito o teste nesse local, mas mais tarde é realocado ao concelho de origem da pessoa em questão. Idem para idades e até género, por testes reportados com dados em falta e corrigidos posteriormente, para satisfazer o resultado do teste em menos de 24 horas.
No entanto há dados que são passiveis de ser actualizados nos dados oficiais e aqui neste repositório - por exemplo as amostras - e outros que são interpretados como verdades absolutas - o PDF diário - e que geraram confusão quando foram actualizados.

No caso dos concelhos, se bem me lembro, havia um erro de falta de reporte (soma de concelhos vs. totais nacionais) significativo desde o início (6 ou 7%) e que subiu a pique a partir de setembro de 2020 com a subida de casos pós-férias. A 26-10 faltavam penso que 20% dos casos, e daí, penso eu, uma das razões para deixarem de os publicar.

Isto tudo para dizer que os dados disponíveis no repositório são os dados oficiais que temos conhecimento da sua existência, e se há dados em falta, ou nunca foram publicados, ou se foram e existem, ajudem-nos a identificar onde estão que arranja-se forma de os ir buscar.

Obrigado.

**Comment by hiimbia** [comment]
Muito obrigada pelo esclarecimento!
Será que me podia explicar melhor o que é o `concelhos_14dias`?

**Comment by davipt** [comment]
Claro.

Vejamos o ultimo PDF [Relatório sexta dia 4](https://github.com/dssg-pt/covid19pt-data/blob/master/dgs-reports-archive/Relatório-de-Situação-732_04_03_2022.pdf) com dados de concelhos até dia 2 inclusivé.

Lisboa é reportada com incidência 1.511 novos casos por cem mil pessoas a 14 dias. Segundo o INE existe 509.515 pessoas em Lisboa. logo se 100 mil são 1.511, 500 mil e picos corresponde a 7.699 novos casos.

O `concelhos_14dias` regista esses 7.699 novos casos nos 14 dias anteriores à data do reporte, 3 de março. No entanto não se sabe quantos novos casos houve em cada um desses dias. Só se pode assumir uma média diária de 550 casos. 

Na semana anterior foram reportado 15.291 casos nos 14 dias anteriores.
Como o valor é de 14 dias mas o relatório é semanal, há uma sobreposição de uma semana.
Digamos Semana 1 e Semana 2 são 15.291, e Semana 2 e Semana 3 são 7.699. Não é fácil saber o valor exacto de cada semana para se tentar adicionar ao acumulado.

Os dados do `concelhos_incidencia` e `concelhos_14dias` correspondem aos valores no `concelhos_new` mas num formato diferente.

`17-02-2022 | LISBOA | 15291 | 1092 | 3001 | [960,Max] | Extremamente Elevado |   |   |   | 15291 | Lisboa e Vale do Tejo | LISBOA | 1106 | 10005.43 | 509515 | 33197 | 33340 | 27706 | 23037 | 26770 | 49807 | 77513 | 110853 | 144050 | 5092.6 | 1439.78 | 1107.98 | 774.74
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --`

`03-03-2022 | LISBOA | 7699 | 550 | 1511 | [960,Max] | Extremamente Elevado |   |   |   | 7699 | Lisboa e Vale do Tejo | LISBOA | 1106 | 10005.43 | 509515 | 33197 | 33340 | 27706 | 23037 | 26770 | 49807 | 77513 | 110853 | 144050 | 5092.6 | 1439.78 | 1107.98 | 774.74
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --`

**Reply by hiimbia** [reply]
Muito Obrigada! Ajudou imenso
